src/styles/_constants.scss

```
@@ -14,20 +14,34 @@ $font-weight-regular: 400;
 $font-weight-medium: 500;
 $font-weight-bold: 700;
 
-// primary font colors
-$color-primary-black: #000;
-$color-primary-white: #fff;
-
-// secondary font colors
-$color-secondary-gray600: #545454;
-$color-secondary-gray500: #757575;
-$color-secondary-gray400: #afafaf;
-$color-secondary-gray200: #e6e6e6;
-$color-secondary-gray100: #f2f2f2;
-$color-secondary-brown: #664d1b;
-$color-secondary-yellow: #ffdb20;
-$color-secondary-yellow-50: #fffaf0;
+
+// COLORS
+
+// Primary colors
+$color-primary-black: hsl(0deg 0% 0%);
+$color-primary-white: hsl(0deg 0% 100%);
+
+// Yellow colors
+$color-secondary-yellow: hsl(50deg 100% 56%);
```

Maybe we still need old color-format for refactoring? E.g:

```suggestion
$color-secondary-yellow: hsl(50deg 100% 56%); // #ffdb20
```